docs/troubleshooting.md

```diff
@@ -0,0 +1,333 @@
+# Troubleshooting
+
+Your AMD sales representative and support team are available to assist with troubleshooting KubernetesOnload
+in your cluster. Please see [release notes](../ReleaseNotes.md).
+
+## FAQ
+
+### How can I confirm Onload Operator has deployed successfully?
+
+[Onload Operator](#onload-operator) resources will be present with their logs clear of errors.
```

Comment:
```suggestion
[Onload Operator](#onload-operator) pod will be running with their logs clear of errors. Onload CRD will have created successfully.
```
Resources could be confusing in the context, since the device-plugin creates a different kind of "resource".